packages/expo-modules-core/ios/Core/Objects/ObjectDefinition.swift

```diff
@@ -68,7 +68,7 @@ public class ObjectDefinition: AnyDefinition, JavaScriptObjectBuilder {
   }
 
   public func decorate(object: JavaScriptObject, appContext: AppContext) throws {
-    let runtime = try appContext.runtime
+    _ = try appContext.runtime
```

Comment:
I thought we might want it for the side effect of throwing if it wasn't available but not sure cc @tsapeta

Reply:
Yeah, it could be like a guard that throws an error earlier, but I think all other decorator functions have similar checks so it's probably not needed here. In other words, I think we can remove it.